api/services/user.go

```diff
@@ -65,21 +78,20 @@ func (s *service) UpdateUser(ctx context.Context, req *requests.UpdateUser) ([]s
 //
 // Deprecated, use [Service.UpdateUser] instead.
 func (s *service) UpdatePasswordUser(ctx context.Context, id, currentPassword, newPassword string) error {
-	user, _, err := s.store.UserGetByID(ctx, id, false)
+	user, err := s.store.UserGet(ctx, store.UserIdentID, id)
```

Comment:
I believe that this duplicated function could die before we do the change. So we have a restore reduction that needs to be supported for the next version.